shell/ev-sidebar-attachments.c

```diff
@@ -565,6 +565,7 @@ ev_sidebar_attachments_init (EvSidebarAttachments *ev_attachbar)
 	gtk_container_add (GTK_CONTAINER (swindow),
 			   ev_attachbar->priv->icon_view);
 
+        gtk_box_pack_start (GTK_BOX (ev_attachbar), swindow, TRUE, TRUE, 0);
```

Comment:
Can you fix the indent please?
Sadly, the code uses tabs here (not our code-style), but you should do the same for one new line.

Reply:
OK, updated my tree, changed the spaces to tabs!